src/formGenerator.vue

```diff
@@ -342,6 +357,11 @@ div.vue-form-generator(v-if='schema != null')
 				return res.map(item => item.error);
 			},
 
+			fieldTouched(field) {
+				let res = this.touched.filter(e => e.field == field);
```

Comment:
Yes sorry, I was tired when I did that and did not bother to write good code. Thank you !